<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferToAccountDialog</class>
 <widget class="QDialog" name="TransferToAccountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>510</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#TransferToAccountDialog{border:1px solid rgb(68,217,199);background-color:white;}
QLineEdit{border:1px solid rgb(214,214,214);border-radius:4px;padding: 0px 8px 0px 8px;font: 12px &quot;微软雅黑&quot;;}
QLabel{font: 12px &quot;微软雅黑&quot;;}</string>
  </property>
  <widget class="QLineEdit" name="sendToLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>340</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>来自：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>86</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QComboBox" name="accountComboBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>340</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{border:1px solid rgb(214,214,214);border-radius:4px;
font: 12px &quot;微软雅黑&quot;;padding: 0px 8px 0px 8px;}
QComboBox::down-arrow {image: url(:pic2/downArrow.png);padding:0px 10px 0px 0px;}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    border-left-width: 0px;
    border-top-right-radius: 4px; /* same radius as the QComboBox */
    border-bottom-right-radius: 4px;
}
QComboBox QAbstractItemView{
border:1px solid rgb(214,214,214);
selection-color:rgb(0,0,0);
outline: 0px;
}
QComboBox QAbstractItemView::item{min-height:30px;}
QComboBox QAbstractItemView::item:selected{background-color:rgb(235,235,241);padding:0px 8px 0px 8px;}
QComboBox QAbstractItemView::item:!selected{background-color:white;padding:0px 8px 0px 8px;}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>来自地址：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="addressLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>340</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>账户余额：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="balanceLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>发往：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>金额：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="amountLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="balanceLabel_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>ALP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>0.01 ALP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>手续费：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>备注：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="remarkLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>340</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>589</x>
     <y>12</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{background-image:url(:pic2/close5.png);background-repeat: repeat-xy;background-position: center;background-attachment: fixed;background-clip: padding;border-style: flat;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>转账</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>长度超过40个字节</string>
   </property>
  </widget>
  <widget class="QToolButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>460</y>
     <width>86</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
